go/libkb/chain_link.go

```diff
@@ -436,6 +438,16 @@ func (tmp *ChainLinkUnpacked) unpackPayloadJSON(payloadJSON *jsonw.Wrapper, payl
 
 	payloadJSON.AtKey("seqno").GetInt64Void(&sq, &err)
 
+	// Assume public unless specified
+	tmp.seqType = keybase1.SeqType_PUBLIC
+	if jw := payloadJSON.AtKey("seq_type"); !jw.IsNil() {
+		seqTypeInt, err := payloadJSON.AtKey("seq_type").GetInt()
+		if err != nil {
+			return err
```

Comment:
erroring out here is slightly scary, if there's some sort of scrod that make it into the chain that we never checked?

Reply:
hate to be so tentative, but also, i don't want to break the app